From 225e393cb6e3e0a47fb60ba7ff436b94d91618e6 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 18 Oct 2021 17:59:44 +0800
Subject: [PATCH] octeontx2-af: fix gain ip failed

Since commit [bc48858bc71d octeontx2-af: refactor function npc_install_flow
for default entry] applied, the macro definition was moved to drivers/net/
ethernet/marvell/octeontx2/af/npc.h.

But the merge commit [4c8aa3594edc Merge branch 'v5.10/standard/base' into
v5.10/standard/cn-sdkv4.18/cn96xx] incorrectly introduced the old definition
again.

Clean it up to fix the issue

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
index b9cf9fb14e6e..97bde0486c80 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_npc.c
@@ -24,10 +24,6 @@
 #define RSVD_MCAM_ENTRIES_PER_PF	3 /* Broadcast, Promisc and AllMulticast */
 #define RSVD_MCAM_ENTRIES_PER_NIXLF	1 /* Ucast for LFs */
 
-#define NIXLF_UCAST_ENTRY	0
-#define NIXLF_BCAST_ENTRY	1
-#define NIXLF_PROMISC_ENTRY	2
-
 #define NPC_PARSE_RESULT_DMAC_OFFSET	8
 #define NPC_HW_TSTAMP_OFFSET		8ULL
 
-- 
2.31.1

